public/css/main.css

```diff
@@ -0,0 +1,17 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    direction: rtl;
+}
+nav{
+    background-color: #333333;
+}
+.nav--content{
+     margin: 0 100px;
+    min-height: 100px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Comment:
@naremanhilles Hi! I recently found out that `justify-content: space-between` can be buggy in Internet Explorer 11.  So my advice (if IE11 support is important to your project, which it might not be at this early phase) is to double-check how it looks in IE11 whenever you use `space-between`.